### PR TITLE
Adding a new entry for array_replace method in php

### PR DESCRIPTION
**Pull request by PriyansuMaurya**
### Issue : #3046
- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-replace/array-replace.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+- `$array` (array): The original array that will be used as the base.
+- `$replacements` (array): One or more arrays containing the replacement values.
+
+> **Note:** If multiple arrays contain the same key, the value from the last array in the argument list will overwrite the previous ones.
+
+## Example
+
+Consider the following arrays:
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-replace/array-replace.md:31]:
```suggestion
The code below demonstrates the basic use of the `array_replace()` function:
```

File: content/php/concepts/arrays/terms/array-replace/array-replace.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+The **`array_replace()`** function in PHP is used to replace the values of an array with values from one or more arrays.
+
+## Syntax
+
+```pseudo
+array_replace(array $array, array ...$replacements): array
+```
+
+- `$array` (array): The original array that will be used as the base.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-replace/array-replace.md:24]:
```suggestion
- `$array` (array): The original array that will be used as the base. The original array will not be modified, a new array will be returned with the associated replacement values.
```

File: content/php/concepts/arrays/terms/array-replace/array-replace.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    [a] => 1
+    [b] => 10
+    [c] => 20
+    [d] => 30
+)
+```
+
+## Codebyte Example
+
+In the following PHP code, the `array_replace()` function is demonstrated by replacing values in the `$baseArray` with values from the `$replacementArray`. The resulting array is then printed.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-replace/array-replace.md:57]:
```suggestion
The following code is runnable and demonstrates other implementation of the `array_replace()` function.
```

**Comment by PriyansuMaurya** [comment]
@caupolicandiaz , I have finish making changes.

**Review by caupolicandiaz — APPROVED** [review]
@PriyansuMaurya There was a missing edit, which I made directly. This is ready for a second review.

File: content/php/concepts/arrays/terms/array-replace/array-replace.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    [a] => 1
+    [b] => 10
+    [c] => 20
+    [d] => 30
+)
+```
+
+## Codebyte Example
+
+The following code is runnable and demonstrates other implementation of the `array_replace()` function:
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-replace/array-replace.md:57]:
```suggestion
The following code is runnable and demonstrates another implementation of the `array_replace()` function:
```

**Review by SSwiniarski — APPROVED** [review]
@PriyansuMaurya Looks good.

**Pull request merged**